src/main/java/it/gov/pagopa/payhub/activities/enums/EntityIdentifierType.java

```diff
@@ -0,0 +1,24 @@
+package it.gov.pagopa.payhub.activities.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum EntityIdentifierType {
+    F("F"),
+    G("G");
+
+    private final String value;
+
+    EntityIdentifierType(String value) {
+
+        this.value = value;
+    }
+
+    public static EntityIdentifierType fromValue(String value) {
+        for (EntityIdentifierType b : EntityIdentifierType.values()) {
+            if (b.value.equals(value)) {
+                return b;
+            }
+        }
+        throw new IllegalArgumentException("Unexpected value '" + value + "'");
+    }}
```

Comment:
if name and value are always equals, all of these is not needed.
Instead, let's add a javadoc to explain values (it's not obvious)
```suggestion
    /** Phisical person */
    F,
    /** Giuridical person */
    G;
}
```